### PR TITLE
remove stale pricing pages

### PR DESCRIPTION
**Pull request by tvald**
Keeping pricing pages in sync was challenging. 

The docs pricing pages had various outdated statements, including mention of $0.50/gb pricing on Topics.

Removing the pages so that we don't have to worry about them any more.

Updated links and redirects as well, so this page depends on the pending redirect PR.

### CODE REVIEW

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md
Comment (malandis) [inline i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md]: FYI the English pricing page is as above but the path is different on the Japanese page (`pricing` vs `price`):

https://www.gomomento.com/pricing

vs

https://jp.gomomento.com/price/

So when I follow the link above to the English page and select Japanese language, I am redirected to the Japanese landing page https://jp.gomomento.com, which I doubt we intend :)

File: i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md
Comment (malandis) [inline i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md:11]: Did you intend to completely remove the link here as opposed to replacing?

**Review by malandis — COMMENTED** [review]
👍 from me after considering a few small issues:

File: i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md
Comment (tvald) [reply i18n/ja/docusaurus-plugin-content-docs/current/topics/index.md:11]: Yes, this sentence was also removed from the English version.

File: i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md
Comment (tvald) [reply i18n/ja/docusaurus-plugin-content-docs/current/cache/introduction/what-is-serverless-caching.md]: Good catch, will fix.

**Comment by malandis** [comment]
Will approve once rebased/merged with the build fix

**Review by malandis — APPROVED** [review]

**Pull request merged**
